pkg/build/builder/util.go

```diff
@@ -150,3 +153,69 @@ func ParseProxyURL(proxy string) (*url.URL, error) {
 
 	return proxyURL, err
 }
+
+// CopyDirIfExists recursively copies a directory to the destination path.
+// If the source directory does not exist, no error is returned.
+// If the destination directory exists, any contents with matching file names
+// will be overwritten.
+func CopyDirIfExists(src, dst string) error {
```

Comment:
Please add unit tests for this function, covering in particular:

1. Source directory not existing
2. Destination already existing (confirm files overwritten
3. Regular files copied
4. Symlinks copied.